src/create_disk.sh

```diff
@@ -61,6 +65,17 @@ udevtrig() {
 export PATH=$PATH:/sbin:/usr/sbin
 arch="$(uname -m)"
 
+# If the platforms table has the legacy flag set for this arch, pretend
+# the table doesn't exist
+if [ -n "$platforms" ] && jq -e ".$arch.legacy" < "$platforms" > /dev/null; then
+    platforms=
```

Comment:
so the legacy flag will be added by default for ppc64le ? the console won't log anything without the console kargs on ppc64le

Reply:
The legacy flag is just for managing the migration to the new behavior, and will go away in some number of weeks.  In non-legacy mode, kola on ppc64le will continue to [inject](https://github.com/coreos/coreos-assembler/pull/2400/files#diff-110657a50e0db550de5910ee3d75e73e9cad0f5f990fb7c6b2cd341cdf1f4951R72) a console argument in `kola run` and `kola qemuexec`, but the images themselves will have no console argument by default.  The assumption is that users can configure a non-default console if they need it.  Are you saying that that behavior is not correct for ppc64le?

Reply:
from the OCP perspective, for baremetal UPI installs, this is totally fine, the console karg can be injected. i'm thinking more about the OCP IPI install scenario where the installer generates the ignition config - maybe for that we can inject console in the ignition through installer/MCO.

Reply:
We want a default console that works for most users, so it'd be helpful to better understand the scope of the problem.  How large is the mismatch between kernel defaults and what's needed?  E.g., if there are no console arguments, by default will the kernel never have a working console?  Have no working console on most hardware?  Create a functioning console on the video card, but ppc64le users always expect to use a serial console?

Reply:
`console=hvc0 console=tty0` is needed on qemu for a working console. for baremetal, i can double check but iirc from a previous conversation with some IBM folks a while ago it is needed for metal too, but I will confirm.

Reply:
after the testing done on baremetal, looks like this is only needed for qemu, so we are all set.

Reply:
Updated https://github.com/coreos/fedora-coreos-config/pull/1181 to apply `console=hvc0 console=tty0` on ppc64le qemu.  Thanks for the research!